### DISCUSSION TITLE
Will this project support other database like mysql or posgresql ?

### DISCUSSION THREAD

**Discussion by mayerui** [fnc12/sqlite_orm#1231] [Ideas]
Will this project support other database like mysql or posgresql ?
What is the rationale for only supporting SQLite ?
It is real lack ORM library in C++.

**Comment by fnc12** [comment]
well this idea is nice. How do you see this in API?

**Comment by xfsaid** [comment]
Looking forward to supporting mysql.

**Comment by fnc12** [comment]
@xfsaid what is the most popular C++ mysql connector lib?

**Comment by fnc12** [comment]
@xfsaid honestly saying we are going to create a new ORM dedicated to MySQL in future. It is going to have closed source and paid license. If you are interested in it let's switch to personal discussion by email fnc12@me.com.

Also I'd like to covert this issue to discussion cause it doesn't have anything to commit into repo
